pkg/utils/utils_test.go

```diff
@@ -1,12 +1,14 @@
 package utils_test
 
 import (
+	"github.com/stretchr/testify/require"
```

Comment:
golangci-lint usually flags these for sorting

Reply:
thought I did fix, but I might have merged after or something, I'll run again, sorry.